src/main/java/com/uber/jenkins/phabricator/conduit/Differential.java

```diff
@@ -119,7 +121,13 @@ public String getSummaryMessage(String phabricatorURL) {
 
     public String getPhabricatorLink(String phabricatorURL) {
         String revisionID = this.getRevisionID(true);
-        return String.format("%s%s", phabricatorURL, revisionID);
+        try {
+            URL base = new URL(phabricatorURL);
+            return new URL(base, revisionID).toString();
+        } catch (MalformedURLException e) {
+            e.printStackTrace();
+            return String.format("%s%s", phabricatorURL, revisionID);
```

Comment:
Making a URL seems like better practice (non-java expert here), but this still doesn't seem to fix #27, at least not on it's own. I think `revisionID.toUpperCase()` would fix it. Or do `str.toUpperCase()` in `getRevisionID()` to ensure that it will always return an uppercase string.